main.go

```diff
@@ -259,6 +259,9 @@ func setupServiceOperatorDefaults() {
 
 	neutronv1.SetupNeutronAPIDefaults(neutronAPIDefaults)
 
+	// Nova
+	novav1.SetupDefaults()
+
```

Comment:
We will replicate this simplified approach for the other operators as we move forward